app/src/androidTest/java/com/github/polypoly/app/menu/ProfileActivityTest.kt

```diff
@@ -0,0 +1,96 @@
+package com.github.polypoly.app.menu
+
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.test.espresso.intent.Intents
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import androidx.test.espresso.intent.Intents.intended
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.intent.matcher.IntentMatchers.*
+import androidx.test.espresso.matcher.ViewMatchers.*
+import com.github.polypoly.app.game.user.allTrophies
+import com.github.polypoly.app.network.FakeRemoteStorage
+
+@RunWith(AndroidJUnit4::class)
+class ProfileActivityTest {
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<ProfileActivity>()
+
+    private val user = FakeRemoteStorage.instance.user
+
+    private val button = composeTestRule.onNodeWithTag("modifyProfileButton")
+
+    @Test
+    fun modifyButtonGoToTheModifyingPage() {
+        Intents.init()
+
+        button.performClick()
+        intended(hasComponent(ProfileModifyingActivity::class.java.name))
+
+        Intents.release()
+    }
+
+    @Test
+    fun canSeeTheDescriptionOfAWonTrophy() {
+        val statisticsAndTrophies = composeTestRule.onNodeWithTag("statisticsAndTrophies")
+        statisticsAndTrophies.onChildren().filter(hasTestTag("Trophy${user.trophiesWon[1]}"))[0]
```

Comment:
- user.trophiesWon[1] could be turned into a variable to improve clarity
- using [0] isn't very explicit, if the idea is to test only one trophy then it should be explained through a comment to make it clearer

Reply:
after the filter, there are only one trophy but in a list, so I put [0] to unwraped it